google/cloud/storage/internal/retry_client_test.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/retry_client.h"
+#include "google/cloud/storage/testing/canonical_errors.h"
+#include "google/cloud/storage/testing/mock_client.h"
+#include "google/cloud/testing_util/chrono_literals.h"
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+namespace {
+using ::testing::_;
+using ::testing::HasSubstr;
+using ::testing::Return;
+using testing::canonical_errors::PermanentError;
+using testing::canonical_errors::TransientError;
+using namespace testing_util::chrono_literals;
```

Comment:
These are a c++14 extension. Do we allow/use >c++11 in unit tests?

Reply:
`std::chrono_literals` are a C++14 feature.  `google::cloud::testing_utils::chrono_literals` are our thing.  So no, we do not allow > C++11 in the unit tests or code[1].


[1]: eventually we need to create build that compile with C++14 and C++17, just to show that it works.

Reply:
Woops, my mistake. I read that too quickly and missed the leading namespace. Sorry about that.